src/core/connectors.js

```diff
@@ -1590,6 +1590,13 @@ const connectors = [{
 	],
 	js: 'connectors/detektorfm.js',
 	id: 'detektorfm',
+}, {
+	label: 'ibroadcast.com',
```

Comment:
It should be iBroadcast, as it's displayed on their website.

Reply:
My bad - saw you fixed it :space_invader: